pytorch_lightning/loops/fit_loop.py

```diff
@@ -263,7 +263,7 @@ def teardown(self) -> None:
 
     def on_save_checkpoint(self) -> Dict:
         state_dict = super().on_save_checkpoint()
-        # FIXME(@tchaton) Should pass has_completed=True when iterator is exhausted ?
+        # TODO: update has_completed to its proper value
```

Comment:
This is not related to this PR, right?

Reply:
nope, but I will take away the responsibility from Thomas :)